client/src/utils/blockstackHelpers.js

```diff
@@ -8,6 +8,17 @@ export const postSingleObservation = async (userSession, observation, observatio
   userSession.putFile(`observation/${fileNumber}.json`, JSON.stringify(observation));
 };
 
+export const deleteAllObservations = async userSession => {
+  await userSession.deleteFile(`observations.json`);
+};
+
+export const deleteSingleObservations = async (userSession, numOfObservations) => {
+  for (let num = 1; num < numOfObservations; num += 1) {
```

Comment:
```suggestion
  for (let num = 1; num <= numOfObservations; num += 1) {
```

Make sure to delete the last observation.